modules/dynamic/test/src/smithy4s/dynamic/DefaultSchematic.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright 2021 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s
+package dynamic
+
+import cats.Id
+import java.util.UUID
+import schematic.ByteArray
+import schematic.Field
+import schematic.Alt
+
+object DefaultSchematic
```

Comment:
This seems useful for working with schemas, maybe it could be exposed to the users in some module closer to core?

Reply:
I thought about, but decided against it because of a lack for a better motivation, at this time. When the time arise, we'll move it